src/App.js

```diff
@@ -1,27 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```

Comment:
- [ ] I would not recommend removing the `App` component. Use the `App.js` as the parent component and it should have the imported `Calculator.js` component.

- [ ] Also, The `App.js` component should be class-based. In this milestone, the objective is to practice the class-based React component.

Reply:
I really removed this file. I am looking for it right now but still, it is not in my project. I think I deleted that file. Could you check if the commit is the most recent, please :pray:
Probably that is why the color is red, I guess.

Thank you for pointing that out, anyways. I learned a lot from this.:+1:
[Check this](https://screenrec.com/share/SIRgV0H9ZT)

Reply:
Yes, you have removed the file. Please add again and do the necessary changes. :smile:

Reply:
Done!:+1:
Pull request resubmitted, also.